lib/popup.ts

```diff
@@ -158,6 +158,14 @@ export class PopupControl<T extends IPopupOptions = IPopupOptions> extends Dispo
   private _showDelay: number = 0;
   private _hideDelay: number = 0;
 
+  constructor() {
+    super();
+    this.onDispose(() => {
+      if (this._openTimer !== undefined) { clearTimeout(this._openTimer); }
+      if (this._closeTimer !== undefined) { clearTimeout(this._closeTimer); }
```

Comment:
Is this the cause of the bug with occasional JS errors? Could you add a comment? (BTW you can just use `if (this._closeTimer)` as in other places in this class)

Reply:
Will do.